### PR TITLE
[WFLY-14563] todo-backend quickstart

### PR DESCRIPTION
**Pull request by jmesnil**
* Updated documentation to be able to generate documentation for EAP XP.

JIRA: https://issues.redhat.com/browse/WFLY-14563

Signed-off-by: Jeff Mesnil <jmesnil@redhat.com>

### CODE REVIEW

File: shared-doc/attributes.adoc [hunk 1]
```diff
@@ -28,7 +28,9 @@ ifdef::ProductRelease[]
 :productNameFull: Red Hat JBoss Enterprise Application Platform
 :productVersion: 7.4
 :DocInfoProductNumber: {productVersion}
-
+:githubRepoUrl: https://github.com/jboss-developer/jboss-eap-quickstarts
```
Comment (emmartins) [inline shared-doc/attributes.adoc:31]: this is defined already a few lines below in the IF product release or xp release, and thus overrides this definition

File: shared-doc/attributes.adoc [hunk 2]
```diff
@@ -37,6 +39,9 @@ ifdef::EAPXPRelease[]
 :productNameFull: Red Hat JBoss Enterprise Application Platform expansion pack
 :productVersion: 3.0
 :DocInfoProductNumber: 7.4
+:githubRepoUrl: https://github.com/jboss-developer/jboss-eap-quickstarts
```
Comment (emmartins) [inline shared-doc/attributes.adoc:42]: this is defined already a few lines below in the IF product release or xp release, and thus overrides this definition

**Review by emmartins — CHANGES_REQUESTED** [review]

File: shared-doc/attributes.adoc (hunk 1 above)
Comment (jmesnil) [reply shared-doc/attributes.adoc]: fixed

File: shared-doc/attributes.adoc (hunk 2 above)
Comment (jmesnil) [reply shared-doc/attributes.adoc]: fixed

**Pull request merged**